src/CommentBox.js

```diff
@@ -50,6 +50,14 @@ class CommentBox extends Component {
     this.loadCommentsFromServer();
     setInterval(this.loadCommentsFromServer, this.props.pollInterval);
   }
+  //when incorporating into another project
+  //(with react-router for instance),
+  //this will prevent error messages every 2 seconds
+  //once the CommentBox is unmounted
+  componentWillUnmount() {
+  this.setInterval && clearInterval(this.setInterval);
```

Comment:
is `this.setInterval` set whenever something with an interval is called? This seems to work fine for this case, but it may be safer to set the interval as its own property on the class on mount. Maybe something like:

```
// constructor
this.pollInterval = null;

// componentDidMount
if (!this.pollInterval) this.pollInterval = setInterval(this.loadCommentsFromServer, this.props.pollInterval);

// componentWillUnmount
this.pollInterval && clearInterval(this.pollInterval);
this.pollInterval = null;
```

I'm not sure, it may be overkill for this situation, but if others end up using this for a reference it may be helpful. What are your thoughts on it?